test/conformance/kuttl/generate/policy/standard/clone/sync/nspol-clone-sync-update/update-generated-resource.yaml

```diff
@@ -0,0 +1,9 @@
+
+apiVersion: v1
```

Comment:
Just to align with naming convention we using with other test directories, can we named the namespace policy dir as `pol-xxx` like `.../clone/sync/pol-clone-sync-update/...` instead of `nspol-xxx`

Reply:
done.

Reply:
@prateekpandey14 could you please review the changes?